torch/csrc/jit/frontend/concrete_module_type.h

```diff
@@ -57,11 +57,14 @@ class ConcreteModuleType;
 // this to decide whether we can share types between modules.
 class VISIBILITY_HIDDEN ConcreteModuleTypeBuilder {
  public:
+  ConcreteModuleTypeBuilder() {}
```

Comment:
Out of curiosity, why does this need to be public now? I don't see any new callsites for it.

Is it due to the changes to `add_constant` in `script_init.cpp`?

Reply:
good catch, it doesn't (I moved this for a different version of this PR and didn't revert)